webpack.config.js

```diff
@@ -0,0 +1,51 @@
+const glob = require(`glob`);
+const path = require(`path`);
+const {EnvironmentPlugin} = require(`webpack`);
+
+module.exports = {
+  entry: `./packages/node_modules/ciscospark`,
+  output: {
+    filename: `bundle.js`,
+    library: `ciscospark`,
+    libraryTarget: `var`,
+    path: __dirname,
+    sourceMapFilename: `[file].map`
+  },
+  devtool: `source-map`,
+  devServer: {
+    https: true,
```

Comment:
Do we need to use HTTPS for localhost?

Reply:
i've found it to be a little hit-or-miss, but in general, yes. various browsers really don't like to let you get camera access if you're not on https.

Reply:
That said, it's mostly just for camera access; i don't think anything else needs it.